indico/modules/events/editing/client/js/editing/timeline/TimelineItem.module.scss

```diff
@@ -5,6 +5,18 @@
 // modify it under the terms of the MIT License; see the
 // LICENSE file for more details.
 
+@import 'base/palette';
+
 .state-indicator {
   margin-left: 5px;
 }
+
+.item-index {
+  color: $gray;
+}
+
+.header > :not(:first-child)::before {
+  content: '\00B7' !important;
+  padding: 0 0.4em;
```

Comment:
I updated this css to provide some padding instead of a special character plus the `{' '}`.

I could have updated it globally, we use it on `_timelines.scss` however I'm not exactly sure of the impact, we would have to update some code and remove the extra spaces.

Reply:
What I don't like with this approach is that you get something like this when copying content from the page (at least in Firefox):

    #1Adrian Mönnich submitted a revisionNovember 12, 2019

Otherwise it'd be a great option!

Reply:
To be completely honest, I would probably still prefer not having the spaces, but I can see it makes sense for some use-cases.